### DISCUSSION TITLE
Disable onboarding for our organisation

### DISCUSSION THREAD

**Discussion by shkm** [renovatebot/renovate#18627] [Request Help]
Hi,

I saw that you're open to disabling the onboarding PR per organisation, as in https://github.com/renovatebot/renovate/discussions/18432. Would you mind applying the same change to our organisation, at https://github.com/nedap?

Thank you!

**Comment by rarkins** [comment]
I have set this in your org config:

```
{
  "onboarding": false
}
```

**Reply by shkm** [reply]
Thanks!
